BuildScripts/mvBESFunctions.sql

```diff
@@ -309,10 +310,12 @@ BEGIN
                                                 || rConst.OPEN_BRACKET || 'log_array' || rConst.COMMA_CHARACTER
                                                 || rConst.SINGLE_QUOTE_CHARACTER || 'log$_' || pOldTableName || rConst.SINGLE_QUOTE_CHARACTER || rConst.COMMA_CHARACTER
                                                 || rConst.SINGLE_QUOTE_CHARACTER || 'log$_' || pNewTableName || rConst.SINGLE_QUOTE_CHARACTER || rConst.CLOSE_BRACKET;
+    tUpdateTriggerName                  := 'ALTER TRIGGER ' || 'trig$_' || pOldTableName || rConst.ON_COMMAND || pNewTableName || rConst.RENAME_TO_COMMAND || 'trig$_' || pNewTableName;
```

Comment:
Nice :+1:
And there's a column inside the mview_logs table that needs this update too